pkg/diagnose/connections.go

```diff
@@ -76,7 +79,47 @@ func Connections(clusterInfo *cluster.Info, _ string, status reporter.Interface)
 	}
 
 	if tracker.HasFailures() {
-		return errors.New("failures while diagnosing connections")
+		return errors.New("failures while diagnosing gateway connections")
+	}
+
+	return nil
+}
+
+func checkRouteAgentConnections(clusterInfo *cluster.Info, status reporter.Interface) error {
+	status.Start("Checking route agent connections")
+	defer status.End()
+
+	routeAgents, err := clusterInfo.GetRouteAgents()
+	if err != nil {
+		return status.Error(err, "Error retrieving route agents")
+	}
+
+	if len(routeAgents) == 0 {
```

Comment:
Do we need to consider if submariner.healthcheck is enabled  ?

Reply:
We create a RouteAgent resource regardless (same with Gatewaya).